account_tax_expense_include/account_tax.py

```diff
@@ -30,3 +30,92 @@ class account_tax(orm.Model):
                                           help="Check this if this tax is \
                                           included in the expense amount."),
     }
+
+    def create(self, cr, uid, vals, context=None):
+        context = context or {}
+
+        if vals.get('expense_include') and vals.get('child_ids'):
+            for child in vals['child_ids']:
+                child[2]['expense_include'] = vals.get('expense_include')
+
+        return super(account_tax, self).create(cr, uid, vals, context=context)
+
+    def write(self, cr, uid, ids, vals, context=None):
+        context = context or {}
+
+        res = super(account_tax, self).write(
+            cr, uid, ids, vals, context=context)
+
+        for tax in self.browse(cr, uid, ids, context=context):
+            for child in tax.child_ids:
+                child.write({'expense_include': tax.expense_include})
```

Comment:
Loop is inefficient. If you can't call write on the record list, call it on the model with the ids, not once per child.